packages/integration-tests/src/package_manager_sanity_checks.test.ts

```diff
@@ -34,10 +34,22 @@ void describe('getting started happy path', async () => {
     // start a local npm proxy and publish the current codebase to the proxy
     await execa('npm', ['run', 'clean:npm-proxy'], { stdio: 'inherit' });
     await execa('npm', ['run', 'vend'], { stdio: 'inherit' });
+    /**
+     * delete .bin folder in the repo because
+     * 1) e2e tests should not depend on them
+     * 2) execa would have package managers to use them if it can not find the binary in the test project
+     */
+    await execaCommand(`rm -rf ${process.cwd()}/node_modules/.bin`);
   });
 
   after(async () => {
+    // TODO: remove the condition once GA https://github.com/aws-amplify/amplify-backend/issues/1013
+    // create-amplify has been installed globally, we need to remove it before running `setup:local`
+    if (packageManager === 'yarn-classic') {
+      await execa('yarn', ['global', 'remove', 'create-amplify']);
+    }
     // stop the npm proxy
+    await execa('npm', ['run', 'setup:local'], { stdio: 'inherit' }); // add tsx back since we removed all the binaries
```

Comment:
Just making sure my previous comment didn't slip through the cracks: can we use `npm install` here instead of `setup:local`?

Reply:
Done 60ef5803e2